robottelo/ui/locators.py

```diff
@@ -641,10 +641,12 @@ def __iter__(self):
         By.XPATH, "//input[@checked='checked']/parent::label"),
     "entity_select": (
```

Comment:
`entity_select` and `entity_deselect` are used by several UI entities.  Can you please run tests for few other entities to make sure this works fine?